kuma/wiki/admin.py

```diff
@@ -363,6 +364,30 @@ def slug_short(self, obj):
         return truncatechars(obj.slug, 25)
     slug_short.short_description = 'Slug'
 
+    def doc_short(self, obj):
+        """
+        Shorten document 'path (name)' representation in list view.
+
+        The important part is having an HTML break character such as a space,
+        so truncating paths to 25 characters after the locale, and long titles
+        at 25 characters.
+        """
+        doc = obj.document
+        if doc:
+            full_path = '/%s/docs/%s' % (doc.locale, doc.slug)
+            if len(full_path) <= 25:
+                path = full_path
+            else:
+                # 25 characters like /en_US/...End/Of/Slug
+                slash2_pos = full_path.index('/', 1) + 1
+                path = '%s...%s' % (full_path[:slash2_pos],
+                                    full_path[(slash2_pos - 22):])
+            title = truncatechars(doc.title, 25)
+            return '%s (%s)' % (path, title)
+        else:
+            return mark_safe('<em>new document</em>')
+    doc_short.short_description = 'Document (if edit)'
```

Comment:
This would be the first translated string (from kuma) in the Django admin in the wiki app, if not over the whole project. Is now the time to cross that line?